redhat-developer/developers.redhat.com
# [RHDX-470] Add Navigation to Layout: two col assembly

### JIRA Issue Link
[RHDX-470](https://projects.engineering.redhat.com/browse/RHDX-470)

### Verification Process
View the /topics/kubernetes page. See the jump to navigation item for Resources.  This navigation item is on Rich Text assembly, because the Two Col assembly doesn't have a navigation field.

- Edit the /topic/kubernetes page
- Remove the placeholder rich text "title" assembly
- Add a Navigation value to the Resources assembly

Verify jump to navigation and the content of the Resources assembly display correctly.
